packages/common/src/styles/theme.ts

```diff
@@ -1,9 +1,10 @@
 import { MixinsOptions as MuiMixinsOptions } from '@material-ui/core/styles/createMixins';
 import {
+  createTheme,
   Theme as MuiTheme,
   ThemeOptions as MuiThemeOptions,
-} from '@material-ui/core/styles';
-import createTheme from '@material-ui/core/styles/createTheme';
+} from '@material-ui/core';
```

Comment:
Again just trying to export everything from `@material-ui/core` was helping me..